machine.go

```diff
@@ -64,6 +66,47 @@ func (machine Machine) isRegistered() bool {
 	return machine.Registered
 }
 
+type MachineAddresses []netaddr.IP
```

Comment:
This type should be called IPAddresses or something else reusable in case it could be useful other places.

Reply:
I have only introduced this type to implement the [Scanner](https://godocs.io/database/sql#Scanner) and [Valuer](https://godocs.io/database/sql#NullBool.Value) interfaces, and consider it a bit of a quick and dirty solution to storing multiple addresses. As can be seen in the other comment about the data model, I consider it more of a temporary state until a more optimal solution comes along("let not perfect be the enemy of good enough" ...), so I am not sure I'd like to see it being introduced elsewhere in the code. :)

Reply:
Fair, I can live with that. I think I am hunted a bit with the fact that we use a relational DB "already" as I would like to just have a state that could be serialised, but thats a bit late now :P. 

Lets called this resolved and work out better solutions in the future.

Reply:
Well, you can always serialize to SQL. sqlite3 is a really nice RDBMS too, and can store everything in a single file. Is there a problem with relational DBs you ran into specifically?

Reply:
Nono, dont get me wrong, I really like sqlite, I just wonder if we could have done with a bit more flexibility in the beginning and then designed an appropriate RDBMS schema later. But we will get there.